django_project/users/models.py

```diff
@@ -1,3 +1,59 @@
 from django.db import models
```

Comment:
Still include this file so you can test the login functionality by creating users through the admin site.